src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header__menu {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__icon {
+  margin-top: 10px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
```

Comment:
Better to avoid height hardcoding, in that case better to set proper paddings instead